packages/plugin-packages/mashroom-websocket/src/backend/webapp/mashroom-bootstrap-webapp.js

```diff
@@ -13,17 +13,19 @@ import type {
     MashroomHttpUpgradeHandler,
     MashroomWebAppPluginBootstrapFunction
 } from '@mashroom/mashroom/type-definitions';
+import TemporaryFileStore from './tempStore';
 
 const bootstrap: MashroomWebAppPluginBootstrapFunction = async (pluginName, pluginConfig, pluginContextHolder) => {
-    const { path, restrictToRoles, enableKeepAlive, keepAliveIntervalSec, maxConnections } = pluginConfig;
+    const { path, restrictToRoles, enableKeepAlive, keepAliveIntervalSec, maxConnections, tmpFileStorePath } = pluginConfig;
     const pluginContext = pluginContextHolder.getPluginContext();
 
     context.restrictToRoles = restrictToRoles;
     context.basePath = path;
     context.enableKeepAlive = enableKeepAlive;
     context.keepAliveIntervalSec = keepAliveIntervalSec;
     context.maxConnections = maxConnections;
-    context.server = new WebSocketServer(pluginContext.loggerFactory);
+    const tmpFileStore = new TemporaryFileStore(tmpFileStorePath);
```

Comment:
You don't pass the second constructor argument here (logger) - therefore you can see a lot of errors like this in the log:

_[2020-10-03T15:28:31.009] [ERROR] mashroom.server - FATAL: Uncaught exception. This is usually a programming mistake. 
Continuing but server might not work as expected. TypeError: Cannot read property 'warn' of undefined
    at ReadFileContext.callback (/Users/jkofler/development/mashroom/packages/plugin-packages/mashroom-websocket/dist/webapp/tempStore.js:52:24)
    at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:261:13)_